src/ambianic/pipeline/ai/fall_detect.py

```diff
@@ -28,17 +28,22 @@ def __init__(self,
         }
         """
         super().__init__(model, **kwargs)
-        # previous pose detection information for frame t-1 and t-2 to compare pose changes against 
-        self._prev_pose_dix = {'t1': [], 't2': []}
-        self._prev_time = {'t1': time.monotonic(), 't2': time.monotonic()-1}
-        self._prev_thumbnail = {'t1': None, 't2': None}
-
-        # angle b/w left shoulder-hip line with vertical axis
-        self._prev_left_angle_with_yaxis = {'t1': None, 't2': None}
-        # angle b/w right shoulder-hip line with vertical axis
-        self._prev_right_angle_with_yaxis = {'t1': None, 't2': None}
         
-        self.previous_body_vector_score = {'t1': 0, 't2': 0}
+        # previous pose detection information for frame at time t-1 and t-2 \
+        # to compare pose changes against
+        self._prev_data = [None] * 2
+
+        _dix = {'_prev_pose_dix': [],
+                '_prev_time': time.monotonic(),
```

Comment:
@bhavikapanara this is not what I asked for. You are still putting constants in strings.

Please read this:
https://softwareengineering.stackexchange.com/questions/367641/does-it-make-sense-to-use-string-constants-in-python-instead-of-string-literals

Reply:
Done